src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -14,6 +14,10 @@ class PDOIntegration extends Integration
 
     const CONNECTION_TAGS_KEY = 'connection_tags';
 
+    const ENGINE_TO_SYSTEM_ARRAY = [
```

Comment:
As far as I understood your code, a better word than "engine" would be "drivers".